front/migrations/db/migration_124.sql

```diff
@@ -0,0 +1,11 @@
+-- Drop existing foreign key constraint
+ALTER TABLE retrieval_documents
+DROP CONSTRAINT retrieval_documents_retrievalactionid_fkey;
```

Comment:
Constraint name is `"retrieval_documents_retrievalActionId_fkey"`

Reply:
Indeed, weird for some reason I had it lowercased on my local 🤷🏼‍♂️